models/result_test.go

```diff
@@ -470,6 +508,72 @@ var _ = Describe("Result", func() {
 		})
 	})
 
+	Describe("#Process", func() {
+		It("update result status to processing", func() {
+			user := FabricateUser(faker.Email(), faker.Password())
+			result := FabricateResult(user)
```

Comment:
Is it possible to pass `status` parameter while creating the result? `result := FabricateResult(user, "pending")`

So that by reading `Expect(result.Status).To(Equal(models.ResultStatusProcessing))` we clearly understand that it updates status from `pending` to ``processing

Reply:
Updated on [5992790](https://github.com/carryall/go-google-scraper-challenge/pull/71/commits/599279085762d7729781d2e5f3ac428fa6f8c16a)